app/models/responses/MdgSub09XiEoriInformationResponse.scala

```diff
@@ -20,12 +20,12 @@ import models.XiEoriAddressInformation
 import play.api.libs.json.{JsPath, Reads}
 import play.api.libs.functional.syntax._
 
-case class MdgSub09XiEoriInformationResponse(xiEori: String, consent: Option[String], address: XiEoriAddressInformation)
+case class MdgSub09XiEoriInformationResponse(xiEori: String, consent: Option[String], address: Option[XiEoriAddressInformation])
```

Comment:
Could you please keep parameters on separate lines as after adding this new parameter, line has become very long?

Reply:
parameters are added to new lines